lib/characters.c

```diff
@@ -1114,6 +1108,5 @@ gregorio_rebuild_first_syllable (gregorio_character ** param_character)
   current_character = *param_character;
   gregorio_go_to_first_character (&current_character);
   (*param_character) = current_character;
-  // this is not working, so people have to mind the way they put text styles on the initial.
-  //gregorio_rebuild_characters (&current_character, CENTER_FULLY_DETERMINED);
 }
+/* vim: set ts=8 sts=2 sw=2 noet: */
```

Comment:
I admit I'm a bit against having editor-specific tags in files... I like the idea of editorconfig, and there's [a plugin for vim](https://github.com/editorconfig/editorconfig-vim#readme). This allows to put preferences once for the whole project, and change them easily... What do you think?